### PR TITLE
Import from Glue DataCatalog feature

### PR DESCRIPTION
**Pull request by jverhoeks**
Adding the ability to import all tables from a Glue Database into a datacontract.
Currently it supports only the aws authorisation from the environment (AWS_PROFILE or access/secret key)

### CODE REVIEW

File: tests/fixtures/glue/datacontract.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+dataContractSpecification: 0.9.3
+id: my-data-contract-id
+info:
+    title: My Data Contract
+    version: 0.0.1
+models:
```
Comment (jochenchrist) [inline tests/fixtures/glue/datacontract.yaml:6]: We could also write the `server` information from the used connection

File: README.md [hunk 2]
```diff
@@ -647,15 +647,15 @@ data products, find the true domain owner of a field attribute)
...
- Usage: datacontract import [OPTIONS]                                                                              
-                                                                                                                   
- Create a data contract from the given source file. Prints to stdout.                                              
-                                                                                                                   
-╭─ Options ───────────────────────────────────────────────────────────────────────────────────────────────────────╮
-│ *  --format        [sql|avro]  The format of the source file. [default: None] [required]                        │
-│ *  --source        TEXT        The path to the file that should be imported. [default: None] [required]         │
-│    --help                      Show this message and exit.                                                      │
-╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+ Usage: cli.py import [OPTIONS]
```
Comment (jochenchrist) [inline README.md:650]: Keep `datacontract` for CLI usage.

**Review by jochenchrist — CHANGES_REQUESTED** [review]

**Comment by jochenchrist** [comment]
Great contribution!

File: tests/fixtures/glue/datacontract.yaml (hunk 1 above)
Comment (jverhoeks) [reply tests/fixtures/glue/datacontract.yaml:12]: Added the Server model, including account, location and name. But Format is part of the table config in Glue. You can have different formats in the same glue database. Leaving the field empty for now.

File: README.md (hunk 2 above)
Comment (jverhoeks) [reply README.md]: Reverted the change

**Review by jochenchrist — APPROVED** [review]

File: tests/test_import_glue.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+logging.basicConfig(level=logging.INFO, force=True)
+
+db_name = "test_database"
+table_name = "test_table"
+
+
+@pytest.fixture(scope="function")
+def aws_credentials():
+    """Mocked AWS Credentials for moto."""
+    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
```
Comment (jochenchrist) [inline tests/test_import_glue.py:21]: Try using `monkeypatch.setenv()`

**Comment by jochenchrist** [comment]
[Pipeline with python 3.12](https://github.com/datacontract/datacontract-cli/actions/runs/8924508043/job/24511368720?pr=166) fails, maybe due to environment settings.

File: tests/test_import_glue.py (hunk 3 above)
Comment (jverhoeks) [reply tests/test_import_glue.py]: I updated the file with monkeypath. Let's hope it works

**Comment by jochenchrist** [comment]
Thanks for fixing.
Happy to merge.

**Pull request merged**